examples/simple_instance/main.tf

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+  region      = "${var.region}"
+}
+
+module "gce-container" {
+  source = "../../../"
+
+  container = {
+    image = "${var.image}"
+  }
+
+  volumes        = []
+  restart_policy = "${var.restart_policy}"
+}
+
+resource "google_compute_instance" "vm" {
+  project      = "${var.project_id}"
+  name         = "${var.instance_name}"
```

Comment:
Just hardcode this.

Reply:
@morgante I've added a hardcoded default to the variable -- in order to keep the testing architecture that I described in my comment below, this needs to remain a variable.